app/com/linkedin/drelephant/analysis/code/impl/AzkabanJarvisCodeExtractor.java

```diff
@@ -0,0 +1,121 @@
+package com.linkedin.drelephant.analysis.code.impl;
+
+import com.linkedin.drelephant.analysis.code.CodeAnalyzerException;
+import com.linkedin.drelephant.analysis.code.CodeExtractor;
+import com.linkedin.drelephant.analysis.code.JobCodeInfoDataSet;
+import com.linkedin.drelephant.clients.azkaban.AzkabanJobStatusUtil;
+import com.linkedin.drelephant.clients.azkaban.AzkabanWorkflowClient;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+import models.AppResult;
+import org.apache.log4j.Logger;
+import org.codehaus.jettison.json.JSONArray;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+
+
+public class AzkabanJarvisCodeExtractor implements CodeExtractor {
```

Comment:
Jarvis is internal to LinkedIn. This should not go to open source ideally. Definitely the URL should be in config.
Can we modify it as a JSON code fetcher where the hierarchy of which node to fetch can be configured.
Or move this code to internal repo.

Reply:
Moved URL to configuration